src/redux/missions/missionsMiddleware.ts

```diff
@@ -12,12 +16,15 @@ interface MissionData {
   };
 }
 
-async function getMissions(callback: (missions: { missions: Phase[] }) => void) {
+async function getMissions(callback: (missions: { missions: Phase[] } | null) => void) {
   const missionTopic = api.startTopic('missions', {});
   const { value } = (await missionTopic.iterator().next()) as MissionData;
-  if (value.missions !== null) {
+  if (value.missions) {
     callback({ missions: value.missions });
+  } else {
+    callback(null);
   }
+
```

Comment:
Can we place the code in the `getMissions` function inside of the `effect`? This is kind of hard to read atm